interface/main/main_screen.php

```diff
@@ -116,6 +116,10 @@
 if ($GLOBALS['new_tabs_layout']) {
```

Comment:
Catch-22.  You could override the global setting here and set the script in tabs mode.  However rest of the scripts - more specifically the standard script running in app mode would not know and will  continue the logic of frames.  That could create some unpredictable situations.